assets/sass/modules/_enroll.scss

```diff
@@ -97,11 +105,11 @@
 .enroll-factor-row-min {
   margin-bottom: 10px;
 
-  h3 {
```

Comment:
why are we getting rid of `h3`?

Reply:
We dont need the padding any more because we are vertically  centering the text using 
align-items: center; in the parent. 

h3 here is the enroll-factor-label

Reply:
so we don't need any padding at the top? I think that was for separating the beacon from the text... Do you mind double checking with a different factor?